redcap_bridge/test_redcap/testfiles/TestProject/project.json

```diff
@@ -13,5 +13,6 @@
     }
   },
   "api_token": "REDCAP_API_TOKEN",
-  "api_url": "https://redcap.int.univ-amu.fr/api/"
+  "api_url": "https://redcap.int.univ-amu.fr/api/",
+  "template_location": "/"
```

Comment:
not sure how this change is related to the topic of the PR.... Is this necessary for the new feature?

Reply:
Mmmh weird because i never go in this file, maybe a wrong manipulation. Sorry for that, but no it's not necessary.

Reply:
So can you revert this?

Reply:
@killianrochet can you undo this so this change is no longer part of this PR?